lib/masamune/actions/date_parse.rb

```diff
@@ -30,7 +30,7 @@ module DateParse
     def parse_datetime_type(key)
       value = options[key]
       Chronic.parse(value).tap do |datetime_value|
-        console("Using '#{datetime_value}' for --#{key}") if value != datetime_value
+        logger.debug("Using '#{datetime_value}' for --#{key}") if value != datetime_value
```

Comment:
this was printing `Using ...` as first line in dump since `console` prints to `STDOUT`